Makefile

```diff
@@ -532,18 +533,26 @@ $(UI_RESOURCES): $(UI_SOURCES) | check-qt-dir
 
 rcc: $(UI_RESOURCES)
 
-TS_SOURCES := $(shell find ui/i18n -iname '*.ts') # ui/i18n/qml_*.ts
-QM_BINARIES := $(shell find ui/i18n -iname "*.ts" | sed 's/\.ts/\.qm/' | sed 's/ui/bin/') # bin/i18n/qml_*.qm
+TS_SOURCE_DIR := ui/i18n
+TS_BUILD_DIR := $(TS_SOURCE_DIR)/build
 
-$(QM_BINARIES): TS_FILE = $(shell echo $@ | sed 's/\.qm/\.ts/' | sed 's/bin/ui/')
-$(QM_BINARIES): $(TS_SOURCES) | check-qt-dir
-	mkdir -p bin/i18n
-	lrelease -removeidentical $(TS_FILE) -qm $@ $(HANDLE_OUTPUT)
+log-update-translations:
+	echo -e "\033[92mUpdating:\033[39m translations"
+
+update-translations: | log-update-translations
+	cmake -S $(TS_SOURCE_DIR) -B $(TS_BUILD_DIR) -Wno-dev $(HANDLE_OUTPUT)
+	cmake --build $(TS_BUILD_DIR) --target update_application_translations $(HANDLE_OUTPUT)
+#	+ cd scripts/translationScripts && ./fixup-base-ts-for-lokalise.py $(HANDLE_OUTPUT)
```

Comment:
This is a manual step (for now) as it requires Python; no change here. We probably don't want or need this to be run everywhere (mac, Windows), can be done only on CI